test/DeblockTest.cc

```diff
@@ -0,0 +1,472 @@
+/*
+ * Copyright(c) 2019 Netflix, Inc.
+ * SPDX - License - Identifier: BSD - 2 - Clause - Patent
+ */
+
+/******************************************************************************
+ * @file DeblockTest.cc
+ *
+ * @brief Unit test for cdef tools:
+ * * aom_lpf_{horizontal, vertical}_{4, 6, 8, 14}_sse2
+ * * aom_lpf_{horizontal, vertical}_{4, 8, 14}_dual_sse2
+ * * aom_highbd_lpf_{horizontal, vertical}_{4, 6, 8, 14}_sse2
+ * * aom_highbd_lpf_{horizontal, vertical}_{4, 8, 14}_dual_sse2
+ *
+ * @author Cidana-Wenyao
+ *
+ ******************************************************************************/
+#include <cmath>
+#include <cstdlib>
+#include <string>
+#include "gtest/gtest.h"
+// workaround to eliminate the compiling warning on linux
+// The macro will conflict with definition in gtest.h
+#ifdef __USE_GNU
+#undef __USE_GNU  // defined in EbThreads.h
+#endif
+#ifdef _GNU_SOURCE
+#undef _GNU_SOURCE  // defined in EbThreads.h
+#endif
+#include "EbDefinitions.h"
+#include "aom_dsp_rtcd.h"
+#include "util.h"
+#include "random.h"
+#include "EbUtility.h"
+#include "EbDeblockingFilter.h"
+#include "acm_random.h"
+#include "EbDeblockingFilter_SSE2.h"
+#include "loopfilter_ref.h"
+
+using ::testing::make_tuple;
+using libaom_test::ACMRandom;
```

Comment:
can it be replaced by SvtRandom?

Reply:
Could, however three different random has to be created since the ranges are different. It's not very  convenient to use SVTRandom in this case.